app/controllers/authorization_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class AuthorizationController < ApplicationController
```

Comment:
we might want to test this

Reply:
Added some tests. There's a stub for the Magic Link email, I'll add that in a subsequent PR.